bookdown/02-basics.Rmd

```diff
@@ -338,11 +338,11 @@ The view is changed in-place though, i.e. the task object itself is modified.
 Just like columns, it is also possible to assign different roles to rows.
 Rows can have two different roles:
 
-1. Role `use`
+1. Role `use`:
```

Comment:
There was no newline in the book. To have newlines we would have to allow to spaces at the end of a line. The `:` might be a compromise.